index.html

```diff
@@ -48,32 +48,221 @@ <h2>I'm a Software Developer</h2>
                                 src="/images/Gitty.png" alt="" /></a>
                     </li>
                     <li>
-                        <a href="https://linkedin.com/in/Mike111222" target="_blank" rel="noopener"> <img class="mysocial"
-                                src="/images/Linkedin.png" alt="" /></a>
+                        <a href="https://linkedin.com/in/Mike111222" target="_blank" rel="noopener"> <img
+                                class="mysocial" src="/images/Linkedin.png" alt="" /></a>
                     </li>
                     <li>
                         <a href="https://www.linkedin.com/company/vivimedia/about/" target="_blank" rel="noopener"> <img
                                 class="mysocial" src="/images/Vic.png" alt="" /></a>
                     </li>
                     <li>
-                        <a href="https://twitter.com/Engr_ImranKhan1" target="_blank" rel="noopener"> <img class="mysocial"
-                                src="/images/Twitter.png" alt="" /></a>
+                        <a href="https://twitter.com/Engr_ImranKhan1" target="_blank" rel="noopener"> <img
+                                class="mysocial" src="/images/Twitter.png" alt="" /></a>
                     </li>
                     <li>
-                        <a href="mailto:tipu870@gmail.com" target="_blank" rel="noopener"> <img class="mysocial" src="/images/Mail.png"
-                                alt="" /></a>
+                        <a href="mailto:tipu870@gmail.com" target="_blank" rel="noopener"> <img class="mysocial"
+                                src="/images/Mail.png" alt="" /></a>
                     </li>
                     ...
                 </ul>
             </div>
             <img class="bgimage" src="/images/Illustration.png" alt="" />
```

Comment:
- [ ] As the prev reviewer mentioned kindly remove this line as it's not part of the content and for aesthetic reasons. You can move it to CSS file.